### PR TITLE
[DONE] force end of BBB meeting if not create or get status 

### PR DESCRIPTION
**Pull request by ptitloup**
Fix issues of BBB Scalite : https://github.com/bigbluebutton/bigbluebutton/issues/18913

Before sending your pull request, make sure the following are done :

* [x] You have read our [contribution guidelines](https://github.com/EsupPortail/Esup-Pod/blob/master/CONTRIBUTING.md).
* [x] Your PR targets the `develop` branch.
* [x] The title of your PR starts with `[WIP]` or `[DONE]`.

### CODE REVIEW

File: pod/meeting/models.py [hunk 1]
```diff
@@ -639,6 +639,8 @@ def create(self, request=None):
         for elt in xmldoc:
             meeting_json[elt.tag] = elt.text
         if meeting_json.get("returncode", "") != "SUCCESS":
+            if self.end():
+                return self.create()
```
Comment (Badatos) [inline pod/meeting/models.py:643]: un petit commentaire serait le bienvenue ;)

**Review by Badatos — CHANGES_REQUESTED** [review]

**Review by LoicBonavent — APPROVED** [review]
Cela me semble correct, mais non testé

File: pod/meeting/models.py [hunk 2]
```diff
@@ -586,12 +586,7 @@ def is_active(self):
         return False
 
     # ##############################    BBB API
-    def create(self, request=None):
-        """Make the url with goods parameters to create the meeting.
-
-        After create the meeting on the BBB instance, call it.
-        """
-        action = "create"
+    def get_meeting_parameters(self):
```
Comment (Badatos) [inline pod/meeting/models.py:589]: manque une petite pydoc

**Review by Badatos — APPROVED** [review]
une petite pydoc et c'est bon ^^

**Pull request merged**